frontend/src/components/admin/question-creation/QuestionEditor.tsx

```diff
@@ -42,23 +60,43 @@ const QuestionEditor = (): React.ReactElement => {
         setShowAddFractionModal,
         showEditorError,
         setShowEditorError,
-        showQuestionPreview,
-        setShowQuestionPreview,
       }}
     >
-      {showQuestionPreview ? (
-        <QuestionPreview />
-      ) : (
-        <>
+      <Switch>
+        <Route
+          exact
+          path={Routes.ASSESSMENT_EDITOR_QUESTION_EDITOR_PAGE({
+            assessmentId: assessmentId && ":assessmentId",
```

Comment:
can these params be set within the `Routes` file?

Reply:
Unfortunately, since we have 3 cases here (create assessment, edit assessment, and render route path), there's no way to do that without making some really ugly code. For example, I tried putting this logic inside the routes file by using `null` for "create" and `undefined` for route paths, but that got super confusing and non-obvious so I reverted it. Unless you think there's huge value in moving it to that file I'd prefer to keep it this way